src/GZip.jl

```diff
@@ -166,39 +172,39 @@ end
 
 # Easy access to gz reading/writing functions (Internal)
 gzgetc(s::GZipStream) =
-    @test_eof_gzerr(s, ccall((:gzgetc, _zlib), Int32, (Ptr{Void},), s.gz_file), -1)
+    @test_eof_gzerr(s, ccall((:gzgetc, _zlib), Int32, (Ptr{Nothing},), s.gz_file), -1)
```

Comment:
We have `Cvoid` for this purpose. (It's an alias for `Nothing`, but it's intended to be used in `ccall` for clarity.)